site/.markdownlint.json

```diff
@@ -16,5 +16,7 @@
   },
   "MD046": false,
   "MD001": false,
-  "MD049": false
+  "MD049": false,
+  "MD055": false,
+  "MD056": false
```

Comment:
So is the precommit and workflow now running for the site directory?

Reply:
yep! Since there's a config in `site`, markdown-cli will pick it up and use it for that directory